components/Header/Header.jsx

```diff
@@ -1,5 +1,7 @@
+import React from "react";
 import styles from "./Header.module.scss";
 import { Link } from "@tanstack/react-location";
+import { Navigation } from "./child_components/Navigation";
```

Comment:
I see you introduce nesting inside the component. However I wonder, could you create child components just next to Header? and then `import from ./Navigation`
That way we can achieve building blocks next to each other, but without continued nesting

Reply:
At the same time. If something is a component, can you export it as default? and `import Navigation....`